src-docs/src/views/progress/progress_example.js

```diff
@@ -30,6 +30,22 @@ const progressFixedSnippet = `<!-- Position at top of parent container -->
 import ProgressSizeColor from './progress_size_color';
 const progressSizeColorSource = require('!!raw-loader!./progress_size_color');
 const progressSizeColorHtml = renderToHtml(ProgressSizeColor);
+const progressSizeColorSnippet = `<EuiProgress
+  value={20}
+  max={100} 
+  size="s"
+  color="accent"
+/>`;
+
+import ProgressChart from './progress_chart';
+const progressChartSource = require('!!raw-loader!./progress_chart');
+const progressChartHtml = renderToHtml(ProgressChart);
+const progressChartSnippet = `<EuiProgress 
+  value={20}
+  valueText={valueText}
```

Comment:
```suggestion
  valueText={true}
```

Consider changing this to `true` instead of `valueText` because I think a lot of people will skip or skim the section above where you explain how `valueText` works and will jump straight to this example thinking they understand it and will reinvent the wheel.